Explorer/Assets/DCL/Notifications/NotificationsController.cs

```diff
@@ -0,0 +1,106 @@
+using CommunicationData.URLHelpers;
+using Cysharp.Threading.Tasks;
+using DCL.Notification.NotificationsBus;
+using DCL.Notification.Serialization;
+using DCL.Web3.Identities;
+using DCL.WebRequests;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using Utility;
+using Utility.Times;
+
+namespace DCL.Notification
+{
+    public class NotificationsController : IDisposable
```

Comment:
Just naming: it doesn't feel descriptive to have one pure pair of View/Controller and another with the prefix `New-`. Maybe we can find better descriptive names based on their responsibilities (just a suggestion)